airflow/contrib/plugins/metastore_browser/main.py

```diff
@@ -71,15 +74,17 @@ def table(self):
         table = m.get_table(table_name)
         return self.render(
             "metastore_browser/table.html",
-            table=table, table_name=table_name, datetime=datetime, int=int)
+            table=table, table_name=table_name, datetime=datetime, int=int,
+            url_prefix=configuration.get_url_prefix())
 
     @expose('/db/')
     def db(self):
         db = request.args.get("db")
         m = HiveMetastoreHook(METASTORE_CONN_ID)
         tables = sorted(m.get_tables(db=db), key=lambda x: x.tableName)
         return self.render(
-            "metastore_browser/db.html", tables=tables, db=db)
+            "metastore_browser/db.html", tables=tables, db=db,
+            url_prefix=configuration.get_url_prefix())
```

Comment:
Instead of passing the url_prefix here can we pass the url_prefix to the `app.register_blueprint()` in airflow/www/app.py? If it works that mean that any user plugins will be registered in the correct prefix without having to have to include this in their code too.

Reply:
Yes it is not useful anymore, I removed it